huey/contrib/djhuey/management/commands/run_huey.py

```diff
@@ -44,19 +44,25 @@ def add_arguments(self, parser):
     def autodiscover(self):
         """Use Django app registry to pull out potential apps with tasks.py module."""
         module_name = 'tasks'
-        for config in django_apps.get_app_configs():
-            app_path = config.module.__path__
-            try:
-                fp, path, description = imp.find_module(module_name, app_path)
-            except ImportError:
-                continue
-            else:
-                import_path = '%s.%s' % (config.name, module_name)
+
+        try:
+            # Django 2.0+
```

Comment:
autodiscover_modules is available from django 1.7 https://github.com/django/django/commit/6feb75129f42fdac751d0b79a2a005b4c0ad5c2d
so I think it could be used directly.

Reply:
I made this in #362